### PR TITLE
Fix UITableView constraints

### PR DESCRIPTION
**Pull request by kenan-mamedoff**

### CODE REVIEW

**Review by gvh — APPROVED** [review]

**Review by dangelovska — APPROVED** [review]

File: brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPTableViewController.swift [hunk 1]
```diff
@@ -63,42 +63,57 @@ class VIPTableViewController<C: CoordinatableRoutes,
...
-            make.leading.trailing.equalToSuperview().inset(Margins.large.rawValue)
-        }
+        topInsetValue = sceneLeftAlignedTitle == nil ? 0 : Margins.extraHuge.rawValue + 28
         
         view.addSubview(tableView)
         tableView.snp.makeConstraints { make in
             make.edges.equalToSuperview()
         }
         
+        tableView.addSubview(leftAlignedTitleLabel)
```
Comment (rcresnik) [inline brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPTableViewController.swift:73]: amm.. whats this?

File: brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPTableViewController.swift [hunk 2]
```diff
@@ -63,42 +63,57 @@ class VIPTableViewController<C: CoordinatableRoutes,
...
+            make.bottom.equalTo(tableView.snp.top)
+            make.height.equalTo(topInsetValue)
+            make.leading.trailing.equalToSuperview().inset(Margins.large.rawValue)
+        }
+        
         tableView.contentInset.top = topInsetValue
     }
     
     func setRoundedShadowBackground() {
+        let small = Margins.small.rawValue
```
Comment (rcresnik) [inline brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPTableViewController.swift:84]: why not use a WrapperView and embed tableView in it.. and then just set shadow/corners on that? 
this seems.. a bit 2much for a VC

File: brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPTableViewController.swift (hunk 1 above)
Comment (kenan-mamedoff) [reply brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPTableViewController.swift:73]: image

Comment (rcresnik) [reply brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPTableViewController.swift:73]: id use large titles for this crap.. imo thats what the designers tried to replicate

Comment (rcresnik) [reply brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPTableViewController.swift:73]: https://images.app.goo.gl/5bwRFvVoLuvGvjCn9

Comment (kenan-mamedoff) [reply brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPTableViewController.swift:73]: Talked to Tina. Apparently they want it as it is in the designs. Also, sometimes, there is a button in front of that title. Seemed easier this way but can be changed.

**Pull request merged**